app/src/main/java/org/mozilla/tv/firefox/telemetry/TelemetryIntegration.kt

```diff
@@ -463,7 +463,7 @@ open class TelemetryIntegration protected constructor(
 
         TelemetryEvent.create(Category.ACTION, Method.RECEIVED_TAB, null)
             .extra(Extra.DEVICE_TYPE, internalDeviceType)
-            .extra(Extra.TOTAL, receivedTabs.urls.size.toString())
+            .extra(Extra.TOTAL, receivedTabs.tabData.size.toString())
```

Comment:
I see why you changed this now... Unfortunately, I told Ashley (and documented) that we'd do the other approach so I'd like to change it back.